v21.2/deploy-lambda-function.md

```diff
@@ -0,0 +1,260 @@
+    -p 9000:8080 init-crdb
+    ~~~
+
+    Note that this command sets the `DATABASE_URL` environment variable to the cluster connection string, where:
+    - `<username>` is the SQL user. By default, this is your {{ site.data.products.db }} account username.
+    - `<password>` is the password for the SQL user. The password will be shown only once in the **Connection info** dialog after creating the cluster.
+    - `<serverless-host>` is the hostname of the Serverless cluster.
+    - `<cluster-name>` is the short name of your cluster plus the tenant ID. For example, `funny-skunk-3`. The `<cluster-name>` is used to identify your tenant cluster on a multitenant host.
+
+    ~~~
+    time="2021-11-15T21:54:31.526" level=info msg="exec '/var/runtime/bootstrap' (cwd=/var/task, handler=)"
+    ~~~
+
+1. (Optional) In a new terminal, test the local Docker container using the [AWS Runtime Interface Emulator](https://docs.aws.amazon.com/lambda/latest/dg/images-test.html):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ curl -XPOST "http://localhost:9000/2015-03-31/functions/function/invocations" -d '{}'
+    ~~~
+
+    ~~~
+    START RequestId: abda8f75-5d04-4864-ab9c-91b1b9384f80 Version: $LATEST
+    [INFO]	2021-11-17T17:38:39.210Z	abda8f75-5d04-4864-ab9c-91b1b9384f80	Hey! You successfully connected to your CockroachDB cluster.
+    [INFO]	2021-11-17T17:38:39.386Z	abda8f75-5d04-4864-ab9c-91b1b9384f80	Created new account with id 422c6b4b-c48c-4d14-992b-631e7beb9367 and balance 321385.
+    [INFO]	2021-11-17T17:38:39.430Z	abda8f75-5d04-4864-ab9c-91b1b9384f80	Created new account with id 6b390ce6-0915-467f-bd16-102665bd5002 and balance 937663.
+    [INFO]	2021-11-17T17:38:39.486Z	abda8f75-5d04-4864-ab9c-91b1b9384f80	Created new account with id d1e94960-912d-4a8e-8b3a-bbc3a71e924f and balance 628323.
+    [INFO]	2021-11-17T17:38:39.535Z	abda8f75-5d04-4864-ab9c-91b1b9384f80	Created new account with id 715a84fc-027a-4cb0-b39d-8d0e030aeb03 and balance 47771.
+    [INFO]	2021-11-17T17:38:39.584Z	abda8f75-5d04-4864-ab9c-91b1b9384f80	Created new account with id da9667ac-6790-4764-8b46-76a5a96516c7 and balance 327718.
+    [INFO]	2021-11-17T17:38:39.641Z	abda8f75-5d04-4864-ab9c-91b1b9384f80	Database initialized.
+    END RequestId: abda8f75-5d04-4864-ab9c-91b1b9384f80
+    REPORT RequestId: abda8f75-5d04-4864-ab9c-91b1b9384f80	Init Duration: 0.56 ms	Duration: 3896.76 ms	Billed Duration: 3897 ms	Memory Size: 3008 MB	Max Memory Used: 3008 M
+    ~~~
+
+## Step 5. Configure AWS
+
+1. Configure the AWS CLI to authenticate with your AWS account:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ aws configure
+    ~~~
+
+    Follow the prompts to authenticate.
+
+    After you configure the CLI, you can use `aws configure` to list out the configuration variables. For example:
```

Comment:
I don't understand the benefit of having users run `aws configure list` and `aws sts get-caller-identity`.